config/README.md

```diff
@@ -1,102 +1,103 @@
-# Multi-region cockroachdb setup
+# Deploying a DSS
+
+## Preface
+
+This doc provides a well-lit path for deploying the DSS and its dependencies (
+namely CockroachDB) on Kubernetes. This is not a requirement, and a DSS node
+can join a cluster as long as it meets the following requirements:
```

Comment:
Since these requirements are referred to from below, let's put them in their own addressable section.  So on this line, "...meets the requirements below." and then add a "## CRDB requirements" heading just below.

Reply:
done